docs/orchestrating-elastic-stack-applications/kibana.asciidoc

```diff
@@ -206,6 +206,18 @@ To deploy more than one instance of Kibana, all the instances must share a same
 * `xpack.reporting.encryptionKey`
 * `xpack.encryptedSavedObjects.encryptionKey`
 
+[TIP]
+====
+If you need to access these encryption keys, you can find them using the `kubectl get secrets` command. The secret is named after the corresponding Kibana instance.
+
+For example, for a Kibana named `my-kibana`, you can run the following command to retrieve the `xpack.security.encryptionKey` key:
+
+[source,shell,subs="attributes"]
+----
+kubectl get secret my-kibana-kb-config -o json | jq -r '.data["kibana.yml"] | @base64d' | yq .xpack.security.encryptionKey
```

Comment:
I don't think installing `yq` is an assumption we make for most of our docs. Personally I would leave it at `grep encryptionKey` to return all three. Maybe like:
```suggestion
kubectl get secret my-kibana-kb-config -o json | jq -r '.data["kibana.yml"] | @base64d' | grep encryptionKey
```